### PR TITLE
the best i can do

### PR DESCRIPTION
**Pull request by Ortiz-Andrew**
Screenshot 2023-09-08 at 12 09 23 AM

### CODE REVIEW

File: css-three-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+.col1 {
+  flex-basis: calc(50% / 3);
+}
+
+.widpic {
+  max-height: auto;
+  width: 150px;
+  flex-basis: 20%;
```
Comment (shawnkost) [inline css-three-column-layout/styles.css]: `width` & `flex-basis` are essentially the same thing (with some caveats). You should not be setting both. You should be using `flex-basis` to essentally set the `width`

File: css-three-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  background-color: white;
+  border-radius: 50%;
+}
+
+.col2 {
+  display: flex;
+  width: 300px;
+  flex-basis: calc(100% / 3);
```
Comment (shawnkost) [inline css-three-column-layout/styles.css]: `width` & `flex-basis` are essentially the same thing (with some caveats). You should not be setting both. You should be using `flex-basis` to essentally set the `width`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Doesn't quite look like the example image Andrew! You need to try to get it as close as possible to the example image. If you are struggling to get that to happen, then you need to make sure to reach out and ask questions, but you can't just turn this in. It should be so close that I can barely notice differences.

For example, the images are being cut off and not showing correctly. Your heading text isn't centered like the example image. It's important that you try and replicate your example image as close as possible, becuase this is what's expected from you on the job. If you need help, then reach out!

**Comment by Ortiz-Andrew** [comment]
Screenshot 2023-09-09 at 7 21 33 PM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**